src/exporters/DRACOExporter.ts

```diff
@@ -47,15 +63,16 @@ DRACOExporter.prototype = {
     let builder
     let dracoObject
 
-    if (geometry.isBufferGeometry !== true) {
+    if (!geometry.isBufferGeometry) {
       throw new Error('THREE.DRACOExporter.parse(geometry, options): geometry is not a THREE.BufferGeometry instance.')
     }
 
-    if (object.isMesh === true) {
+    if (object instanceof Mesh && object.isMesh) {
       builder = new dracoEncoder.MeshBuilder()
       dracoObject = new dracoEncoder.Mesh()
 
-      var vertices = geometry.getAttribute('position')
+      const vertices = geometry.getAttribute('position')
+      /// @ts-expect-error
```

Comment:
Functions and methods that do not exist in the type are marked with `@ts-expect-error`, although there are several others